cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxAudioFocusManager.java

```diff
@@ -114,13 +154,6 @@ static void unregisterAudioFocusListener(Context context) {
         } else {
             Log.e(TAG, "abandonAudioFocus failed!");
         }
-
-        Cocos2dxHelper.runOnGLThread(new Runnable() {
-            @Override
-            public void run() {
-                nativeOnAudioFocusChange(AUDIOFOCUS_GAIN);
-            }
-        });
```

Comment:
Why remove it?

Reply:
UnregisterAudioFocusListener will be invoked when app on pause, and audio focus will lose, so I think it doesn't need to gain the audio focus.